governance/meeting-host-playbook.md

```diff
@@ -0,0 +1,84 @@
+# TAG Environmental Sustainability Meeting Host Playbook
+
+Meetings are held to coordinate and discuss TAG ENV activities. The TAG ENV leads host these meetings. The following playbook can be used for reference to structure the meeting, you don't need to stick step by step (word by word), it should give guidance how a meeting usually is structured.
+
+## Requirements to host the meeting
+
+* Install Zoom and check your settings (especially audio and video).
+* Get access to the [CNCF TAG YouTube Channel](https://www.youtube.com/@CNCFEnvTAG) to start the livestream.
+
+## Pre-Meeting
+* Add a new blank agenda entry to the meeting notes by copying the template and editing the details accordingly.
+* Announce the meeting about an hour before in the `#tag-env-sustainability` Slack channel.
+
```

Comment:
When possible, maybe make sure the agenda is filled out at least 24 hours ahead of time with some work, so time isn't wasted.  There may be cases in working groups where a delay is needed, and trying to minimize people's time is helpfu.l

Reply:
Good point! Added